gluten-core/src/main/scala/io/glutenproject/expression/ExpressionConverter.scala

```diff
@@ -17,19 +17,44 @@
 
 package io.glutenproject.expression
 
+import io.glutenproject.GlutenConfig
 import io.glutenproject.backendsapi.BackendsApiManager
 import io.glutenproject.execution.{NativeColumnarToRowExec, WholeStageTransformerExec}
-
+import io.glutenproject.expression.ConverterUtils.VELOX_EXPR_BLACKLIST
 import org.apache.spark.internal.Logging
 import org.apache.spark.sql.catalyst.expressions._
 import org.apache.spark.sql.execution._
 import org.apache.spark.sql.execution.exchange.BroadcastExchangeExec
 import org.apache.spark.sql.types.DecimalType
 
 object ExpressionConverter extends Logging {
-  def replaceWithExpressionTransformer(
-                                        expr: Expression,
-                                        attributeSeq: Seq[Attribute]): Expression =
+
+  def doValidate(blacklist: Map[String, String], expr: Expression): Boolean = {
+    val value = blacklist.get(expr.prettyName.toLowerCase())
+    if (value.isEmpty) {
+      return true
+    }
+    val inputTypeName = value.get
+    if (inputTypeName.equals("")) {
+      return false
+    } else {
+      for (input <- expr.children) {
+        if (inputTypeName.equals(input.dataType.typeName)) {
+          return false
+        }
+      }
+    }
+    true
+  }
+
+  def doValidate(expr: Expression): Boolean =
+    GlutenConfig.enableVeloxBackend() && !doValidate(VELOX_EXPR_BLACKLIST, expr)
```

Comment:
Would sth. like `GlutenConfig.getConf.isVeloxBackend` work?

Reply:
Yes, it works and I've updated.